cmd/list.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"sort"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/turbot/steampipe/pkg/cmdconfig"
+	"github.com/turbot/steampipe/pkg/constants"
+	"github.com/turbot/steampipe/pkg/display"
+	"github.com/turbot/steampipe/pkg/error_helpers"
+	"github.com/turbot/steampipe/pkg/steampipeconfig/modconfig"
+	"github.com/turbot/steampipe/pkg/workspace"
+)
+
+type listSubCmdOptions struct {
+	shortDescription string
```

Comment:
can we not just have fixed values for the descriptions and NOT have them in the options?